stable/openstack-base/README.md

```diff
@@ -183,7 +183,7 @@ You'll need to import an image into Glance in order to create instances.
 
 First download a boot image, like Focal amd64:
 
-    curl http://cloud-images.ubuntu.com/focal/current/focal-server-cloudimg-amd64.img
+    curl http://cloud-images.ubuntu.com/focal/current/focal-server-cloudimg-amd64.img \
```

Comment:
Can you please do the same fixes in at least the following files? Thanks!

* development/openstack-base-hirsute-wallaby/README.md
* development/openstack-base-impish-xena/README.md

Reply:
Added to both of them.